bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -297,6 +298,71 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
         _deleteContent: function (event) {
             event.content.delete({api: this.get('host').get('capi')}, function () {});
         },
+
+        /**
+         * changeLanguage event handler. It opens languageSelectionBox for selecting
+         * language of created content
+         *
+         * @method _selectLanguage
+         * @param e {Object} event facade
+         */
+        _selectLanguage: function (e) {
+            e.preventDefault();
+            this.get('host').get('app').fire('languageSelect', {
```

Comment:
You can store `host` in the variable to not invoke `this.get('host')` multiple times. You can do the same in `_setLanguage` method.